reference/replica-operations.md

```diff
@@ -2,27 +2,26 @@
 
-If a nexus is restarted, i.e. the Mayastor pod hosting it restarts, with the assistance of the control plane it will 'recompos' itself; all of the previous member replicas will be re-attached to it, if possible, including any that were in the state `CHILD_FAULTED`.  If faulted replicas can be re-connected successfully, then the control plane will attempt to rebuild them directly, rather than seek replacements for them first.  This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If they are successfully re-attached but then continue to encounter I/O failures the rebuild will fail and will not be attempted again.
+If a nexus is restarted, i.e. the Mayastor pod hosting it restarts, with the assistance of the control plane it will 'recompose' itself; all of the previous healthy member replicas will be re-attached to it.  If previously faulted replicas are available to be re-connected (`Online`), then the control plane will attempt to reuse and rebuild them directly, rather than seek replacements for them first. This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If the rebuild fails then we follow the above process of removing a `Faulted` replica and adding a new one.
 
-Once provisioned, the replica count of a volume can be changed by editing its corresponding MayastorVolume \(MSV\) custom resource object.  The value of the `Replica Count` field may be either increased or decreased and the control plane will attempt to reconcile the desired and actual state, following the same replica placement rules as described herein.  If the replica count is reduced, faulted replicas will be selected for removal in preference to healthy ones.
+Once provisioned, the replica count of a volume can be changed using the kubectl-mayastor plugin `scale` subcommand.  The value of the `num_replicas` field may be either increased or decreased and the control plane will attempt to reconcile the desired and actual state, following the same replica placement rules as described herein.  If the replica count is reduced, faulted replicas will be selected for removal in preference to healthy ones.
 
-Whenever attempting to reconcile the replica count of an existing volume, should the control plane fail to locate an eligible pool for a required replica, or should the replica creation call fail, the reconciliation 'loop' for that Volume will cease; the desired vs actual replica count will remain diverged.
+Whenever attempting to reconcile the replica count of an existing volume, should the control plane fail to locate an eligible pool for a required replica, or should the replica creation call fail, the reconciliation 'loop' for that Volume will keep retrying at a set interval (30s by default).
```

Comment:
Reading this paragraph after the previous suggests that, if increasing/decreasing the number of replicas fails, then the reconcile loop continually tries to perform this operation in the background (which is not the case). 

I know we are talking about something slightly different here but I wonder if there is a better way to make this obvious to the reader.

Reply:
Yeah, I think we just need to word it differently on the previous paragraph which is when the operation is undone and not retried. Once the operation succeeds and we have a failure for some reason then we do retry forever.